client/src/tokens/typography.js

```diff
@@ -155,13 +155,13 @@ const typeScale = {
     color: 'colors.primary.DEFAULT',
     lineHeight: 'lineHeight.tight',
   },
-  'w-body-text': {
-    fontSize: 'fontSize.16',
+  'w-body-text-large': {
+    fontSize: 'fontSize.18',
     fontWeight: 'fontWeight.normal',
     lineHeight: 'lineHeight.normal',
   },
-  'w-body-text-large': {
-    fontSize: 'fontSize.18',
+  'w-body-text': {
+    fontSize: 'fontSize.16',
```

Comment:
Is this re-ordering required?

Reply:
It is my proposal: I moved the 'large body text' element up as the other elements are in descending order, for better consistency.
Before implementing it, I discussed it with Thibaus https://github.com/wagtail/wagtail/issues/8659 and received an approval.

Reply:
makes sense, thank you.